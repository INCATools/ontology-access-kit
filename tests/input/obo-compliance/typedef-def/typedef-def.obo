format-version: 1.4
ontology: typedef-def

[Typedef]
id: R:1
def: "An example definition." []

