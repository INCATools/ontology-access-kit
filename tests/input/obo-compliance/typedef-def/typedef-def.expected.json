{
  "graphs" : [ {
    "id" : "http://purl.obolibrary.org/obo/typedef-def.owl",
    "meta" : {
      "basicPropertyValues" : [ {
        "pred" : "http://www.geneontology.org/formats/oboInOwl#hasOBOFormatVersion",
        "val" : "1.4"
      } ]
    },
    "nodes" : [ {
      "id" : "http://purl.obolibrary.org/obo/IAO_0000115",
      "lbl" : "definition",
      "type" : "PROPERTY"
    }, {
      "id" : "http://purl.obolibrary.org/obo/R_1",
      "type" : "PROPERTY",
      "meta" : {
        "definition" : {
          "val" : "An example definition."
        }
      }
    }, {
      "id" : "http://www.geneontology.org/formats/oboInOwl#hasOBOFormatVersion",
      "lbl" : "has_obo_format_version",
      "type" : "PROPERTY"
    }, {
      "id" : "http://www.geneontology.org/formats/oboInOwl#id",
      "lbl" : "id",
      "type" : "PROPERTY"
    } ]
  } ]
}